src/core/api/public_api.ts

```diff
@@ -1014,6 +1023,16 @@ class Player extends EventEmitter<IPublicAPIEvent> {
     return manifest.isLive;
   }
 
+  /**
+   * Returns true if both:
+   *   - a content is loaded
+   *   - the content loaded is a low latency content
+   * @returns {Boolean}
+   */
+  isLowLatency() : boolean {
```

Comment:
Why?
In the demo you can already know.

Also: `the content loaded is a low latency content`
Not really, it just says that the `lowLatencyMode` option was set